server/models/boxCrime.js

```diff
@@ -0,0 +1,104 @@
+var db = require('../../database/index').db;
+var BoxCrime = require('../../database/models/BoxCrime');
+
+var storeOpenDataInBoxes = (crimeData) => {
+  
+  var asyncBoxStore = crimeData.forEach((crime) => {
```

Comment:
Interesting. It seems like it stored all the data, since I was able to query the database later. I played around with the fiddle to double check, and a simple swap from forEach to map should work just fine 👍